src/lib/helpers/config.ts

```diff
@@ -5,6 +5,9 @@ export const ConfigSchema = z.object({
     server: z.object({
         port: z.number().default(Number(Deno.env.get("PORT")) || 8282),
         host: z.string().default(Deno.env.get("HOST") || "127.0.0.1"),
+        base_path: z.string().default(
+            Deno.env.get("SERVER_BASE_PATH") || "/companion",
+        ),
```

Comment:
```suggestion
        base_path: z.string().regex("^/").default(
            Deno.env.get("SERVER_BASE_PATH") || "/companion",
        ),
```
Might be worth considering adding a quick regex check here to ensure it's prefixed with a forward-slash. Without it (i.e. `base_path = "companion"` in the toml), companion starts up fine but doesn't work in a non-obvious way:

![image](https://github.com/user-attachments/assets/2a4a4dd4-413e-4cdc-b569-536f74a6203a)

The suggestion is untested.